Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: popup panel stacking

### Description (*)

This PR simplifies PopupPanel so its only responsibility is to display a single message. It moves the responsibility of creating those messages to the PopupManager. The PopupManager will instantiate PopupPanel gameObjects and display them when it is called. It will reuse those instantiated objects when possible (which is most of the time since we likely won't display more than one or two at once in a regular case). 
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT2733](https://jira.unity3d.com/browse/MTT-2733)
### Manual testing scenarios

The simplest way to see this PR's effects is to duplicate calls to ShowPopupPanel() to have multiple popups at once. Or add a coroutine that perdically calls this, so that you can experiment with closing some of them before new ones are created.
### Questions or comments

### Contribution checklist
